### ISSUE TITLE
Blastvisu need Biopython

### ISSUE THREAD

**Issue by lecorguille** [lecorguille/galaxy-blastvisu#2]
During Emma intership, the visualisation was a tool and works well.
During Caroline intership, the visualisation become a real visualisation and work well on her setup.

But on a production Galaxy instance, Biopython is needed but seems not available

```
Module blastvisu_mako:68 in render_body         view
>>  from Bio.Blast import NCBIXML
ImportError: No module named Bio.Blast
```

So, we will need to find a way:
- install Biopython
- use an alternative to parse the xml file

**Comment by bgruening** [comment]
I guess you need to have this installed into the Galaxy venv to make it work.
@guerler is it possible to have a requirement tag inside the visualisation.xml file? This way we could hook up arbitrary dependencies in visualisations.

**Comment by lecorguille** [comment]
Indeed, my next move will be to install Biopython in the Venv. But I was surprise because the user galaxy can access to the lib Biopython installed on the cluster.
I haven't search so much. 

And sure, it would be great if there is a system of tool dependenties.
